docker-entrypoint.sh

```diff
@@ -45,13 +51,12 @@ formatGeneratedFiles() {
 }
 
 yeoman_target_dir="/network/target"
+fabrica_config_path="../../network/fabrica-config.json"
+yeoman_command=${1:-setup-docker}
+yeoman_param=${2:-"$fabrica_config_path"}
 
-if [ -n "$1" ]; then
-  executeYeomanCommand "$yeoman_target_dir" "$1" "$2"
-else
-  # by default entrypoint generates files from fabrica config
-  config_path="/network/fabrica-config.json"
+executeYeomanCommandAndCleanup "$yeoman_command" "$yeoman_param"
 
-  executeYeomanCommand "$yeoman_target_dir" "setup-docker \"../..$config_path\""
-  formatGeneratedFiles "$yeoman_target_dir"
+if [ "$yeoman_command" = setup-docker ]; then
+  formatGeneratedFiles
```

Comment:
hm... a czy to zawsze zadziała? Widze scenariusz pt. :
ustawiliśmy tls na false, a orderera w trybie raft.
i teraz kiedy zrobimy setup-docker to validacja nie przejdzie, ale poleci do formatowania plików ktorych nie ma